DOC-814: Document inline_boundaries setting changes (#1946)
tinymce/tinymce-docs: release-notes/release-notes58.md

The following new features were added for the {{site.productname}} 5.8 release.

### Improved keyboard navigation for the inline boundary feature

> Users can now escape inline elements (such as links and comments) using the `Home`, `End`, `PageUp`, and `PageDown` keys. Due to their unique behavior, `PageUp` and `PageDown` are not available in Mac os.

Reviewer: ```suggestion
Users can now escape inline elements (such as links and comments) using the `Home`, `End`, `PageUp`, and `PageDown` keys. Due to their unique behavior, `PageUp` and `PageDown` are not available in macOS.
```

The following new features were added for the {{site.productname}} 5.8 release.

> ### Add support for keys Home, End, PageUp and PageDown to escape inline elements

Reviewer 2: ```suggestion
### Improved keyboard navigation for inline elements
```

Reviewer: ### Improved keyboard navigation for the inline boundary feature

> A new set of keys has been added `Home`, `End`, `PageUp`, and `PageDown` to allow escape outside of the inline element scope (eg for links or comments).

Reviewer 2: ```suggestion
Users can now escape inline elements; such as links and comments; using the `Home`, `End`, `PageUp`, and `PageDown` keys.
```

> 

Reviewer: We need a "see" more link here that links to the inline_boundary setting.

Reviewer 2: https://github.com/tinymce/tinymce-docs/pull/1946/files#diff-5c6bf60f64c3e3eec017e2e616a6fc0a8ce325aaea57e18f156c04eb1fcf5595L103 is an example of this

Author: updated

> Users can now escape inline elements (such as links and comments) using the `Home`, `End`, `PageUp`, and `PageDown` keys, see: [Content appearance options - Inline boundaries]({{site.baseurl}}/configure/content-appearance/#inline_boundaries).

Reviewer 2: Need to explain why we are linking to that other page. Since this functionality isn't documented there, we nee to tell them why we have the link. We link there so we don't have to re-explain inline boundaries here in the release notes. Unfortunately, we don't explain what we mean by "inline boundaries" anywhere in the docs atm.

For example:
```suggestion
Users can now escape inline elements (such as links and comments) using the `Home`, `End`, `PageUp`, and `PageDown` keys.

For information on inline boundaries, see: [Content appearance options - Inline boundaries]({{site.baseurl}}/configure/content-appearance/#inline_boundaries).
```

> Users can now escape inline elements (such as links and comments) using the `Home`, `End`, `PageUp`, and `PageDown` keys, see: [Content appearance options - Inline boundaries]({{site.baseurl}}/configure/content-appearance/#inline_boundaries).

Reviewer: Also do we need to mention this isn't applicable to Mac for page up/down? as Macs define a different behaviour for page up/down.
